docker-compose.yml

```diff
@@ -38,7 +38,7 @@ services:
         sed -i '/# Primary VT build/i add_definitions( \
           \"-Xclang -load \
           -Xclang /serialization-sanitizer/libsanitizer-plugin.so \
-          -Xclang -plugin -Xclang sanitizer\" \
+          -Xclang -add-plugin -Xclang sanitizer\" \
```

Comment:
clang would not generate an object file for `vt_git_revision.cc` when used with `plugin` (it works for all other files)